libs/x-content/build.gradle

```diff
@@ -58,12 +58,6 @@ tasks.named('forbiddenApisMain').configure {
   replaceSignatureFiles 'jdk-signatures'
 }
 
-thirdPartyAudit.ignoreMissingClasses(
```

Comment:
Do we need to touch these files in other plugins folders?

Reply:
This was required to be removed as precommit task would fail otherwise.